apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -6,15 +6,19 @@ import useTopPostsQuery from '../hooks/use-top-posts-query';
 
 import './hightlights.scss';
 
-function TopColumn( { items, viewAllUrl, viewAllText, title, className = null } ) {
+function TopColumn( { items, viewAllUrl, viewAllText, title, isLoading, className = null } ) {
 	const translate = useTranslate();
 
 	return (
 		<div className={ classNames( 'stats-widget-highlights-card', className ) }>
 			<label className="stats-widget-highlights-card__title">{ title }</label>
 			{ items.length === 0 && (
 				<div className="stats-widget-highlights-card__empty">
-					<span>{ translate( 'Sorry, nothing to report.' ) }</span>
+					<span>
+						{ isLoading
+							? `${ translate( 'Loading' ) }...`
+							: translate( 'Sorry, nothing to report.' ) }
+					</span>
```

Comment:
Is this `span` necessary?

Reply:
I prefer to avoid putting text inside the div tag directly. However, wrapping it with the p tag looks more reasonable. 🙂

Reply:
Hmm... What's your rationale for avoiding text nodes directly within div's?